CLAUDE.md

```diff
@@ -0,0 +1,23 @@
+# CLAUDE.md
+
+This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
+
+## Build/Lint/Test Commands
+- Install and update: `rush update`
+- Build: `rush build` (all), `rush build -t <packagename>` (single), `rushx build` (current)
+- Lint: `rushx lint` (check), `rushx lint:fix` (fix), `rushx format` (prettier)
+- Test: `rushx test` (all), `rushx test:node`, `rushx test:browser`
+- Single test: `rushx test:vitest -- -t "test name pattern"`
```

Comment:
I don't think we have "test:vitest" script.  Should this be "rushx unit-test:node" (most used) or just "rushx unit-test"?

Reply:
Fixed